src/gui/DatabaseTabWidget.cpp

```diff
@@ -748,11 +748,13 @@ void DatabaseTabWidget::performGlobalAutoType()
 
     // TODO: allow for database selection during Auto-Type instead of using the current tab
     if (!unlockedDatabases.isEmpty()) {
-        autoType()->performGlobalAutoType(unlockedDatabases);
+        autoType()->performGlobalAutoType(unlockedDatabases, url);
     } else if (count() > 0) {
         if (config()->get(Config::Security_RelockAutoType).toBool()) {
             m_dbWidgetPendingLock = currentDatabaseWidget();
         }
+
+        currentDatabaseWidget()->setUrlForAutoType(url);
```

Comment:
I didn't want to mess up any unlock signals with the URL parameter, so here's a workaround.